stdlib/hash.gr

```diff
@@ -121,8 +121,8 @@ let rec hashOne = (val, depth) => {
   } else if ((val & Tags._GRAIN_NUMBER_TAG_MASK) != 0n) {
     hash32(val)
   } else if (
-    (val & Tags._GRAIN_GENERIC_TAG_MASK) ==
-    Tags._GRAIN_GENERIC_HEAP_TAG_TYPE
+    (val & Tags._GRAIN_GENERIC_TAG_MASK)
+    == Tags._GRAIN_GENERIC_HEAP_TAG_TYPE
```

Comment:
Reading through all the formatted code, I find it looks really wierd with the `==` on a newline mostly because its not often that you chain equals together I wonder if we should preserve the old formatting style for `==`.